gdal/doc/source/drivers/raster/cog.rst

```diff
@@ -183,13 +192,14 @@ Reprojection related creation options
   units of TARGET_SRS. Only taken into account if TARGET_SRS is specified.
 
 - **ALIGNED_LEVELS=INT**: Number of overview levels for which GeoTIFF tile and
-  WebMercator tiles match. When specifying this option, padding tiles will be
+  tiles defined in the tiling scheme match. When specifying this option, padding tiles will be
   added to the left and top sides of the target raster, when needed, so that
-  a GeoTIFF tile matches with a tile of the GoogleMapsCompatible tiling scheme.
-  Only taken into account if TILING_SCHEME=GoogleMapsCompatible. As up to
-  2^ALIGNED_LEVELS tiles can be added in each dimension, it is the responsibility
-  of the user to use this setting with care (a hard limit of 10 is enforced by
-  the driver).
+  a GeoTIFF tile matches with a tile of the tiling scheme.
+  Only taken into account if TILING_SCHEME is different from CUSTOM.
+  For a tiling scheme whose resolution of consecutive zoom levels differ by
```

Comment:
Maybe this sentence is better worded as "For a tiling scheme whose consecutive zoom level resolutions differ by a factor of 2, care must be taken in setting this value to a high number of levels, as up to 2^(ALIGNED_LEVELS-1) tiles can be added in each dimension. The driver enforces a hard limit of 10."

Reply:
Thanks. Integrated